nexus-repository-helm/src/main/java/org/sonatype/repository/helm/HelmAttributes.java

```diff
@@ -36,11 +37,31 @@ public HelmAttributes(final Map<String, Object> attributesMap) {
     attributesMap.forEach((key, value) -> {
       Optional<HelmProperties> propertyOpt = HelmProperties.findByPropertyName(key);
       if (value != null && propertyOpt.isPresent()) {
-        attributesEnumMap.put(propertyOpt.get(), value);
+        putInMap(propertyOpt.get(), value);
       }
     });
   }
 
+  private void putInMap(final HelmProperties helmProperties, final Object value) {
+    switch (helmProperties){
+      case MAINTAINERS:
+        // convert map: name: Bitnami
+        //              email: containers@bitnami.com
+        //              name: wbuchwalter
+        //              email: wibuch@microsoft.com
+        // in list: Bitnami : containers@bitnami.com , wbuchwalter : wibuch@microsoft.com
+        List<Map<String, String>> maintainersMap = (List<Map<String, String>>) value;
+        List<String> maintainers = maintainersMap.stream()
```

Comment:
Question: By default any attributes are stored as nested key-value map. Can storing attributes this way break anything? Like search, for example?

Reply:
I think it's better to ask someone from community about such cases

Reply:
I think that it depends on search feature implementation, I fixed just representing of this attribute. Anyway as I can see we cant save in NastedAttribute any Map (check NestedAttributesMap.set() implementation)